### DISCUSSION TITLE
Missing documentation for `debugFactory`

### DISCUSSION THREAD

**Discussion by emulienfou** [loopbackio/loopback-next#8315] [Ideas]
Would be nice to have a section related to `debugFactory` in the **Troubleshooting** section of the documentation.

* The only "example" and place where it is mentioned is in this next page: https://loopback.io/doc/en/lb4/Accessing-http-request-response.html#use-middleware-to-access-http-requestresponse
* The other place where it's seems to be used in some example is in the shopping one: https://github.com/loopbackio/loopback4-example-shopping

Here is a simple example on how to use the `debugFactory`:
```
import debugFactory from 'debug';

// Define debug factory pattern
const debug = debugFactory('example:debug:factory');

debug('Hello debug message');
```

To debug app with custom pattern use: `DEBUG=example:debug:factory npm start`

**Comment by achrinza** [comment]
Thanks for the suggestion!

I suppose it's assumed that LoopBack 4 developers would already have a preference on which library they'd use for debug message. `debug` is just what's used internally in LoopBack 4, and there's no mandate for LoopBack 4 users to use the same library. Though documenting how and why we use it would a good resource for those starting out.

If you'd like to help document this, I'm more than happy to review your PR.

**Reply by emulienfou** [reply]
Fixed by PR https://github.com/loopbackio/loopback-next/pull/8344
